examples/llama.py

```diff
@@ -189,6 +189,11 @@ def __call__(self, tokens:Tensor, start_pos:int, temperature:Optional[float]=Non
     },
   },
   "2": {
+    #TinyLLaMA https://github.com/jzhang38/TinyLlama
```

Comment:
cool! i think it needs to have its own entry and not under "2" since it's not official llama 2